### ISSUE TITLE
[Feature Request]: Add metronome 

### ISSUE THREAD

**Issue by aquada** [jithware/brethap#75]
It would be nice to have a tick sound (every second) to get an idea when the end of a breath/hold is complete, and maybe the sound adaptive to the time, eg as you breathe out the sound winds down until the end. The [prana breath app ](https://play.google.com/store/apps/details?id=com.abdula.pranabreath&gl=US) is an example of this
thank you

**Comment by jithware** [comment]
Please provide the tick audio file as documented in https://github.com/jithware/brethap/issues/57#issuecomment-1238629942. Also include a README.md file in the same directory that describes the source of the audio file and/or method of creation if it is an original work.

Once the above is complete, I'll look into adding it to the app.

**Comment by jithware** [comment]
Added instructions for audio file inclusion see: https://github.com/jithware/brethap/discussions/76
